src/services/commit/useCommitComponents.tsx

```diff
@@ -47,19 +47,20 @@ const query = `
   ) {
     owner(username: $owner) {
       repository(name: $repo) {
+        __typename
         ... on Repository {
           commit(id: $commitId) {
             components (filters: $filters) {
               name
             }
           }
         }
-      }
-      ... on NotFoundError {
-        message
-      }
-      ... on OwnerNotActivatedError {
-        message
+        ... on NotFoundError {
```

Comment:
Thanks. I was taking a look at this as well. Can we get a demo of the selector working as intended in the description?